assets/css/app.scss

```diff
@@ -1,6 +1,7 @@
 /* This file is for your main application css. */
 @import 'bourbon';
 @import 'neat';
+@import "../../deps/phoenix_live_view/assets/css/live_view.css";
```

Comment:
Prefer single quoted strings

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#stringquotes)